src/typescript/13-utility-types/utility-types.md

```diff
@@ -1,17 +1,17 @@
 @page learn-typescript/utility-types Utility Types
 @parent learn-typescript 13
+@outline 3
 
 @description Learn About the Utility Types provided by TypeScript
 
 @body
 
 ## Overview
 
-Utility Types are built-in helper Types meant for common Type transformations.
+In this section, you will:
 
-In this section you will see a small description for the most used Utility Types.
-
-We are going to repeat the interface Dinosaur below throughout the examples.
+- Review popular utility types
+- We are going to repeat the interface Dinosaur below throughout the examples.
```

Comment:
I kept the code to inform the user that we are keeping the same example. Do we need it though?

Reply:
No, I think that can be repeated in the files and we don’t need it at the top.